packages/frontend-design-poc/src/App.tsx

```diff
@@ -1,25 +1,25 @@
-import { Route, Routes } from "react-router-dom";
-import { HelloWorld } from "./components/HelloWorld";
-import { PageNotFound } from "./pages/PageNotFound";
-import { PageLayout } from "./pages/PageLayout";
-import { Inbox } from "./pages/Inbox";
-import { InboxItemPage } from "./pages/InboxItemPage";
+import { Route, Routes } from 'react-router-dom';
+import { HelloWorld } from './components/HelloWorld';
+import { PageNotFound } from './pages/PageNotFound';
+import { Inbox } from './pages/Inbox';
+import { InboxItemPage } from './pages/InboxItemPage';
+import { Home } from './pages/Home';
 
-import styles from "./app.module.css";
+import styles from './app.module.css';
 
 function App() {
-  return (
-    <main className={styles.app}>
-      <Routes>
-        <Route element={<PageLayout />}>
-          <Route path="/" element={<HelloWorld />} />
-          <Route path="/inbox" element={<Inbox />} />
-          <Route path="/inbox/:id" element={<InboxItemPage />} />
-          <Route path="*" element={<PageNotFound />} />
-        </Route>
-      </Routes>
-    </main>
-  );
+	return (
+		<main className={styles.app}>
```

Comment:
I think we should keep PageLayout as top level Layout component.
Home on the other hand doesn't make sense to have above PayLayout, but as a separate Page / Route component (similar to inibox), mounted on /.

I.e.:

- PageLayout (contains header, main / outlet, footer, etc)
       -  / :  Replace HelloWorld with Home
       - /inbox: <Inbox />
       etc.